iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/clients/client/newclientsecretshow/newclientsecretshow.component.html

```diff
@@ -0,0 +1,65 @@
+<!--
+
+    Copyright (c) Istituto Nazionale di Fisica Nucleare (INFN). 2016-2021
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<div class="modal-header">
+    <h3 class="modal-title">{{ $ctrl.data.title }}</h3>
+</div>
+<div class="modal-body">
+    <p>{{ $ctrl.data.message }}</p>
+    <p>client secret</p>
+
+    <div ng-if="$ctrl.client.registration_access_token && !$ctrl.isLimited()">
+        <div class="input-group">
+            <span class="input-group-btn">
+                <button class="btn btn-default" type="button" ngclipboard data-clipboard-target="#rat"
+                    ngclipboard-success="$ctrl.clipboardSuccess(e);" ngclipboard-error="$ctrl.clipboardError(e);"
+                    alt="Copy registration access token to clipboard">
+                    <i class="fa fa-clipboard"></i>
+                </button>
+            </span>
+            <input type="text" id="rat" ng-model="$ctrl.client.registration_access_token" class="form-control" readonly>
+        </div>
+    </div>
+
+    <div class="input-group" ng-if="$ctrl.confirmation">
+        <span class="input-group-btn" ng-if="!$ctrl.showSecret">
+            <button class="btn btn-default" type="button" ng-click="$ctrl.toggleSecretVisibility()">
+                <i class="fa fa-eye"></i>
+            </button>
+        </span>
+        <span class="input-group-btn" ng-if="$ctrl.showSecret">
+            <button class="btn btn-default" type="button" ngclipboard data-clipboard-target="#client_secret"
+                ngclipboard-success="$ctrl.clipboardSuccess(e, 'secret');" ngclipboard-error="$ctrl.clipboardError(e);"
+                alt="Copy secret to clipboard">
+                <i class="fa fa-clipboard"></i>
+            </button>
+        </span>
+        <input type="password" value="################" ng-if="!$ctrl.showSecret" class="form-control" readonly>
```

Comment:
Why the `value="################"`?
Is supposed to be a `placeholder="***********"`?